inductiva/tasks/task.py

```diff
@@ -96,7 +96,7 @@ def __init__(self, **kwargs):
         self.computation_start_time = None
         self.computation_end_time = None
         self.end_time = None
-        self.cost = None
+        self.estimated_cost_per_hour = None
```

Comment:
The JSON response parameter is actually `estimated_computation_cost`.

Reply:
Fixed it